hummingbot/data_feed/candles_feed/candles_factory.py

```diff
@@ -0,0 +1,14 @@
+from hummingbot.data_feed.candles_feed.binance_perpetuals_candles import BinancePerpetualsCandles
+from hummingbot.data_feed.candles_feed.binance_spot_candles import BinanceSpotCandles
+
+
+class CandlesFactory:
+
+    @classmethod
+    def get_candle(cls, connector: str, trading_pair: str, interval: str = "1m", max_records: int = 500):
+        if connector == "binance_perpetuals":
```

Comment:
Can you change the possible connector values to `binance_perpetual` and `binance`? This will make the options identical to the connector names that users use for the `connect` command.